### ISSUE TITLE
typo in /bin/configure-gtk

### ISSUE THREAD

**Issue by styts** [smravec/nixos-config#4]
I'm using your config as a base of my config - thanks, your is nice and lean.

I'm new to nixos, but this seems to be a typo (maybe it wouldn't work otherwise, idk).

https://github.com/smravec/nixos-config/blob/f32f40ad80b38222493df36b4e59d79484e44def/nixos/wayland/window-manager.nix#L18

`destination = "/bin/configure-gtk";`

**Comment by smravec** [comment]
Yeah that is definitely a typo and fixing it should not break anything (kinda suprised it still worked as I thought there cant be subdirectories in /bin). When I get to my pc I will fix it, or if you want to submit a PR until then I would merge it :)
